internal/test/wmcb_test.go

```diff
@@ -41,6 +41,8 @@ var windowsTaint = v1.Taint{
 	Effect: v1.TaintEffectNoSchedule,
 }
 
+var nodeLabel = "node.openshift.io/os_id=Windows"
```

Comment:
may be combining all the variable declarations at a single place, looks more uniform..?.
Also the doc missing for the variable.